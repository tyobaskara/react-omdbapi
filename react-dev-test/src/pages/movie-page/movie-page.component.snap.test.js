import React from 'react';
import renderer from 'react-test-renderer';
import MoviePage from './movie-page.component';

jest
  .mock('../../components/movie-detail/movie-detail.component', () => 'MovieDetail')
  .mock('../../components/spinner/spinner.component', () => 'Spinner');

test('MoviePage snaptest', () => {
  const props = {
    imdbID: '123'
  };
  const component = renderer.create(
    <MoviePage {...props} />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
