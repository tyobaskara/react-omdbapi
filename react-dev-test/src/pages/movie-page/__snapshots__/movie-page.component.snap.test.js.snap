// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePage snaptest 1`] = `
<div
  className="movie-page"
>
  <MovieDetail
    detailData={Object {}}
  />
</div>
`;
